typescript/src/events/guilds/members/rawMemberRemoveNotify.ts

```diff
@@ -13,8 +12,9 @@ import { Guild, GuildMember, MessageEmbed } from 'discord.js';
 @ApplyOptions<EventOptions>({ event: Events.RawMemberRemove })
 export class UserEvent extends Event {
 	public async run(guild: Guild, member: GuildMember | null, { user }: GatewayGuildMemberRemoveDispatch['d']) {
-		const [enabled, t] = await guild.readSettings((settings) => [settings[GuildSettings.Events.MemberRemove], settings.getLanguage()]);
-		if (!enabled) return;
+		const key = GuildSettings.Channels.Logs.MemberRemove;
+		const [channelId, t] = await guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

Comment:
```suggestion
		const [logChannelId, t] = await guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.